src/main/java/com/faforever/client/mod/ModServiceImpl.java

```diff
@@ -242,7 +246,7 @@ public void enableSimMods(Set<String> simMods) throws IOException {
 
   @Override
   public boolean isModInstalled(String uid) {
-    return getInstalledUiModsUids().contains(uid) || getInstalledModUids().contains(uid);
+    return getInstalledModUids().contains(uid);
```

Comment:
Are we sure that we don't need check the UI mods as well? Both are mods. Otherwise, we need to rename the method. And I expect that this method is being used from the mod vault as well, so now UI mods will probably never been detected as installed.

Reply:
getInstalledUiModsUids is obsolute because all UI Mods are also in the getInstalledMods....

Reply:
it is not necessary to check if it is in the UI Mods because all UI mods are also mods.... See the code of both methods and you will notice ;)